academic-services/knowledge-exploration-service/reference-get-histogram.md

```diff
@@ -0,0 +1,223 @@
+---
+title: GET Histogram
+description: Computes attribute value distribution for a structured query expression
+ms.topic: reference
+ms.date: 2020-02-07
+---
+
+# Histogram REST API
+
+The **Histogram** method computes the entities matching a structured query expression and then calculates the distribution of the requested attributes values in the matched entities.
+
+This method is useful for determining the most dominant attribute values in a result set, e.g. finding the most dominant conference Microsoft publishes in (see [examples](#examples) section below).
+
+``` HTTP
+GET http://{serviceName}.{serviceRegion}.cloudapp.azure.com/calchistogram?expr={expr}
+```  
+
+With optional parameters:
+
+``` HTTP
+GET http://{serviceName}.{serviceRegion}.cloudapp.azure.com/calchistogram?expr={expr}&attributes={attributes}&offset={offset}&count={count}
+```  
+
+## URI Parameters
+
+Name | Required | Type | Description
+--- | --- | --- | ---
+`expr` | Required | string | Structured query expression to evaluate for matching entities used in generating histogram. See [structured query expressions](concepts-query-expressions.md) for documentation.
+`attributes` | Optional | string | A list of comma-separated attributes to generate histograms for. See the [entity schema](reference-makes-api-entity-schema.md) for the attributes that can be requested. If no attributes are specified the response will still include the total number of matching entities. <br/><br/>Defaults to an empty string.
+`offset` | Optional | integer | The number of histogram values to skip. <br/><br/>Defaults to 0.
+`count` | Optional | integer | The number of histogram values to return for each attribute. <br/><br/>Defaults to 10.
+
+## Responses
+
+Name | Type | Description
+--- | --- | ---
+200 OK | [HistogramResponse](#histogramresponse) | Histogram response successfully generated and returned.
+
+## Definitions
+
+| | |
+| --- | --- |
+[HistogramAttribute](#histogramattribute) | Information for an individual attribute
+[HistogramAttributeValue](#histogramattributevalue) | Information for a distinct attribute value
+[HistogramResponse](#histogramresponse) | Histogram response information
+
+### HistogramAttribute
```

Comment:
>HistogramAttribute [](start = 4, length = 18)

missing "distinct_values" in HistogramAttribute

Reply:
It is no longer supported

---
In reply to: [373263208](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/97#discussion_r373263208) [](ancestors = 373263208)